<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DumpWidget</class>
 <widget class="QWidget" name="DumpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupSchema">
     <property name="title">
      <string>Schema</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="chkTables">
        <property name="text">
         <string>Tables</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkViews">
        <property name="text">
         <string>Views</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkTriggers">
        <property name="text">
         <string>Triggers</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkProcedures">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Procedures</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkFunctions">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Functions</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupData">
     <property name="title">
      <string>Data</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="chkLimit">
        <property name="text">
         <string>Dump at max this many table rows:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinLimit">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10000000</number>
        </property>
        <property name="singleStep">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupCompression">
     <property name="title">
      <string>Compression</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QRadioButton" name="radioNoCompression">
        <property name="text">
         <string>No compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioZip">
        <property name="text">
         <string>Zip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioGzip">
        <property name="text">
         <string>Gzip</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBzip2">
        <property name="text">
         <string>Bzip2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutSave">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>&amp;Save</string>
       </property>
       <property name="icon">
        <iconset resource="icons.qrc">
         <normaloff>:/16/document-save</normaloff>:/16/document-save</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupProgress">
     <property name="title">
      <string>Progress</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QProgressBar" name="mainProgressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="subProgressBar"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>groupSchema</tabstop>
  <tabstop>chkTables</tabstop>
  <tabstop>chkViews</tabstop>
  <tabstop>chkProcedures</tabstop>
  <tabstop>chkFunctions</tabstop>
  <tabstop>groupData</tabstop>
  <tabstop>chkLimit</tabstop>
  <tabstop>spinLimit</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>chkLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>spinLimit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>174</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>178</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
